### PR TITLE
Add .NET 5 support

### PR DESCRIPTION
**Pull request by coderpatros**

### CODE REVIEW

**Comment by coderpatros** [comment]
@slang25 @spboyer RFC on .NET 5 support (please ignore failing tests, it looks like GitHub workflows are having some issues running atm)

I have previously thought, for this and other tools I maintain, that only LTS versions of .NET should be supported. But I don't really want to go 2 years and have to make the jump from 3.1 to 6. So I'm proposing that supported versions should be current LTS and latest.

**Comment by spboyer** [comment]
LGTM

File: src/DotNetOutdated/DotNetOutdated.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <LangVersion>7.1</LangVersion>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5</TargetFrameworks>
```
Comment (slang25) [inline src/DotNetOutdated/DotNetOutdated.csproj:5]: `net5.0`, and elsewhere

Comment (coderpatros) [reply src/DotNetOutdated/DotNetOutdated.csproj]: Good pick up.

**Pull request merged**